docs/sql-server/fabric-database/fabric-mirrored-databases.md

```diff
@@ -22,24 +22,24 @@ For more information, see:
 
-With Mirroring in Fabric, you don't need to piece together different services from multiple vendors. Instead, you can enjoy a highly integrated, end-to-end, and easy-to-use product that is designed to simplify your analytics needs, and built for openness and collaboration between technology solutions that can read the open-source Delta Lake table format.
+With mirroring, you don't need to piece together different services from multiple vendors. Instead, you can enjoy a highly integrated, end-to-end, and easy-to-use product that is designed to simplify your analytics needs, and built for openness and collaboration between technology solutions that can read the open-source Delta Lake table format.
 
-## Fabric mirrored databases technology
+## The technology within mirrored databases
 
-The Fabric mirrored database feature uses similar change feed technology as the Azure Synapse Link, and shares some system objects.
+The mirroring feature uses similar change feed technology as the Azure Synapse Link, and shares some system objects.
 
 > [!NOTE]
-> Enabling Mirroring via the Fabric portal will create a `changefeed` database user, a `changefeed` schema, and several tables within the `changefeed` schema in your source database. Do not alter any of these system-managed objects.
+> Enabling mirroring via the Fabric portal will create a `changefeed` database user, a `changefeed` schema, and several tables within the `changefeed` schema in your source database. Do not alter any of these system-managed objects.
 
-## SQL database in Microsoft Fabric
+## Mirrored SQL database
 
-You can also create an SQL database in Microsoft Fabric. With [SQL database in Fabric](/fabric/database/sql/overview), your data is automatically accessible from other Fabric experiences. SQL database in Microsoft Fabric, which uses the same SQL Database Engine as Microsoft SQL Server and is similar to Azure SQL Database, inherits most of the Fabric mirroring capabilities from Azure SQL Database. Your SQL database in Fabric is automatically mirrored to OneLake and presented in a read-only, queryable format. You can use all the different services in Fabric, such as running analytics with Spark, executing notebooks, data engineering, visualizing through Power BI Reports, and more.
+You can also create a mirrored SQL database in Microsoft Fabric. With [SQL database in Fabric](/fabric/database/sql/overview), your data is automatically accessible from other Fabric experiences. SQL database in Microsoft Fabric, which uses the same SQL Database Engine as Microsoft SQL Server and is similar to Azure SQL Database, inherits most of the Fabric mirroring capabilities from Azure SQL database. Your SQL database in Fabric is automatically mirrored to OneLake and presented in a read-only, queryable format. You can use all the different services in Fabric, such as running analytics with Spark, executing notebooks, data engineering, visualizing through Power BI Reports, and more.
```

Comment:
This line is pointing out that you can also create a SQL database in Microsoft Fabric, which is distinct from a mirrored database item. Good catch on the typo tho.